data/mur_citation_map.csv

```diff
@@ -0,0 +1,172 @@
+2:431,52:30101
```

Comment:
@adborden I think this is generated because on uscode it has a complete map for all of US2 to US52.  However, on http://www.fec.gov/law/feca/feca.pdf it focuses only on the FECA (parts of US 52 the FEC administers).

The duplicate arises because 2:431 note can either map to 52:10101 or 52:30101 when taken as a whole.  But FEC will only ever use 30101.